### ISSUE TITLE
Does .newtab-site organic need same styles?

### ISSUE THREAD

**Issue by marcoscaceres** [mozilla/remote-newtab#62]
We currently have:

``` css
.newtab-site:-moz-any([data-type=enhanced], [data-type=sponsored], [data-suggested=true]) .newtab-sponsored {
```

Does `data-type=organic` need to be included in the above?

**Comment by sarracini** [comment]
TIL: tiles can be of type sponsored + suggested, enhanced + suggested, OR organic + suggested. So yes!

**Comment by sarracini** [comment]
this was fixed in #65 ... we can close this too
